src/components/FormFooter/index.tsx

```diff
@@ -3,7 +3,7 @@ import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router-dom';
 import { Button, Icon } from '@trussworks/react-uswds';
 
-type FormFooterProps = {
+type SubmittionFooterProps = {
```

Comment:
This has nothing to do with my code, but making these changes to address type errors